nathan-jacob/scripts/article.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var articles = [];
+
+function Article (rawDataObj) {
+  this.author = rawDataObj.author;
+  this.authorUrl = rawDataObj.authorUrl;
+  this.title = rawDataObj.title;
+  this.category = rawDataObj.category;
+  this.body = rawDataObj.body;
+  this.publishedOn = rawDataObj.publishedOn;
+}
+
+Article.prototype.toHtml = function() {
+  var source = $('#handlebars').html();
+  var template = Handlebars.compile(source);
+  var someArticle = new Article(this);
+  var html = template(someArticle);
+  $('#articles').append(html);
```

Comment:
You do not need to append `html` to the DOM since this method is called inside `append` on line 44.